sdk/communication/Azure.Communication.CallingServer/src/Models/StartRecordingOptions.cs

```diff
@@ -1,13 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
+
 namespace Azure.Communication.CallingServer
 {
     /// <summary>
-    /// Optional parameters for the Start Recording operation.
+    /// Options for the Start Recording operation.
     /// </summary>
     public class StartRecordingOptions
     {
+        /// <summary>
+        /// Parameters for the Start Recording operation.
+        /// </summary>
+        /// <param name="callLocator"> . </param>
+        public StartRecordingOptions(CallLocator callLocator)
+        {
+            CallLocator = callLocator ?? throw new ArgumentNullException(nameof(callLocator));
+        }
+
+        /// <summary>
+        /// The callLocator.
+        /// </summary>
+        internal CallLocator CallLocator { get; }
+
+        /// <summary>
+        /// The callLocator.
+        /// </summary>
+        public Uri RecordingStateCallbackEndpoint { get; set; }
```

Comment:
remove the setter from this.

Reply:
As discussed, the callback Uri doesn't seem to be required on the PMA side